atm.js

```diff
@@ -16,6 +16,35 @@ function savingsDeposit(amount){
 
 function savingsWithdrawal(amount){
 
+}*/
+
+// an eventListerner for each button, each one a "click"
+function checkingBalance(){
+  this.entries = [];
+  this.els = {
+    form: $("#entry"),
+    input: $("#newEntry"),
+    total: $("#total"),
+    entries: $("#entries")
+  }
 }
 
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+checkingBalance.prototype.calculateTotal = function(){
+  var total = 0;
+  for (var i = 0; i < this.entries.length; i++){
+    total += this.entries[i];
+  }
+  return total;
+}
+
+var checkingDeposit = new checkingBalance();
+checkingDeposit.els.form.on("submit", function(event){
+  event.preventDefault();
+  var newDeposit = parseFloat(checkingDeposit.els.input.val())
+  console.log(newDeposit)
+  checkingDeposit.els.input.val("")
+  checkingDeposit.entries.push(newDeposit);
+  var newBalance = checkingDeposit.calculateTotal()
+    checkingDeposit.els.total.html("$" + newBalance)
+
+})
```

Comment:
Looks like a good start logic wise, but would recommend breaking down each step as it applies to this assignment. 

What are the element we care about? 
How could we select and store those elements so as to be able to access those values later?
What functionality is shared across both accounts?